src/main/java/tsunami/security/scanner/utilities/App.java

```diff
@@ -16,12 +16,180 @@
 
 package tsunami.security.scanner.utilities;
 
+import com.beust.jcommander.JCommander;
+import io.kubernetes.client.custom.IntOrString;
+import io.kubernetes.client.openapi.ApiClient;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.Configuration;
+import io.kubernetes.client.openapi.apis.AppsV1Api;
+import io.kubernetes.client.openapi.apis.CoreV1Api;
+import io.kubernetes.client.openapi.models.*;
+import io.kubernetes.client.util.ClientBuilder;
+import io.kubernetes.client.util.Config;
+import io.kubernetes.client.util.KubeConfig;
+import io.kubernetes.client.util.Yaml;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class App {
-    public String getGreeting() {
-        return "Hello world.";
-    }
+    public static void main(String[] args) throws IOException, ApiException, ClassNotFoundException {
+        /* Parse args read from command line */
+
+        ApplicationArgs jArgs = new ApplicationArgs();
+        JCommander helloCmd = JCommander.newBuilder()
+                .addObject(jArgs)
+                .build();
+        helloCmd.parse(args);
+
+        String appName = jArgs.getName();
+        String appVersion = jArgs.getVersion();
+
+        System.out.println("Hello " + appName + " " + appVersion);
+
+        /* Logic to get the relative application config */
+        String configPath = System.getProperty("user.dir") + "/application/" + appName + "/";
+        File file = new File(configPath);
+        if (file.isDirectory()) {
+            try {
+                File[] files = file.listFiles();
+                for (File f : files) {
+                    if (!f.isDirectory())
+                        /* TODO: parse all config files under a certain app name to Kubernetes Object here, maybe use index to represent each config file*/
+                        System.out.println(f);
+                }
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
```

Comment:
You can simplify this code with `MoreFiles.fileTraverser()` from Guava. Guava is the common library released by Google with a lot of useful utilities that can make you life easier. See the document for this util [here](https://guava.dev/releases/29.0-jre/api/docs/com/google/common/io/MoreFiles.html#fileTraverser--).